LibreTransmitter/LibreTransmitterManagerV3.swift

```diff
@@ -276,7 +275,18 @@ public final class LibreTransmitterManagerV3: CGMManager, LibreTransmitterDelega
     var lastDirectUpdate: Date?
 
     internal var countTimesWithoutData: Int = 0
-    
+
+    open var pairingService: SensorPairingProtocol {
+        return SensorPairingService()
+    }
+
+    open var bluetoothSearcher: BluetoothSearcher {
```

Comment:
This creates a new bluetothsearcher every time this variable is accessed. It is now referenced by the SettingsView. So each time the settingsview is created, a new bluetoothsearcher is also created.

Reply:
You can see in the logs, when settingsview appears "BluetoothSearchManager init called" will be logged as well

Reply:
Since the searching is associated with a UI experience, this seems like it could be reasonable.  Is there an issue? Is SettingsView being recreated more than once each time a user goes to scan?

Reply:
It creates an unnecessary bluetooth scan every time an enduser accesses  the Settings View (during the normal runtime of the sensor, long time after pairing is completed)

Reply:
Looking into this, it doesn't look to me like a scan would happen when a user accesses the SettingsView. The BluetoothSearchManager init will be called, but `scanForCompatibleDevices` doesn't happen until the BluetoothSelection view is onscreen.  Is something else triggering a scan, or is this ok?